File: src/main/java/net/ddns/tiojack/htte/model/Player.java
package net.ddns.tiojack.htte.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

import java.io.Serializable;

@Builder
@Value
@Jacksonized
public class Player implements Serializable {

    private static final long serialVersionUID = 7365552274125573163L;

    int playerId;
    double form;
    double stamina;
    double keeper;
    double defender;
    double playmaker;
    double winger;
    double passing;
    double scorer;
    double setPieces;
    double experience;
    double leadership;
    double loyalty;
    boolean motherClubBonus;
    Specialty specialty;
    int age;
    int days;
    int inclusionWeek;
    int daysForNextTraining;

    @JsonIgnore
    public double getSkill(final Skill skill) {
        switch (skill) {
            case GOALKEEPING:
                return this.keeper;
            case DEFENDING:
                return this.defender;
            case WINGER:
                return this.winger;
            case PLAY_MAKING:
                return this.playmaker;
            case SCORING:
                return this.scorer;
            case PASSING:
                return this.passing;
            case SET_PIECES:
                return this.setPieces;
        }
        return this.setPieces;
    }

}

File: src/main/angular/src/model/Player.ts
import { IPlayer } from './IPlayer';

export class Player implements IPlayer {
  playerId: number;
  name: string;
  label: string;
  age: number;
  days: number;
  form: number;
  stamina: number;
  leadership: number;
  experience: number;
  loyalty: number;
  motherClubBonus: number;
  specialty: number;
  keeper: number;
  defending: number;
  playmaking: number;
  winger: number;
  passing: number;
  scoring: number;
  setPieces: number;
  inclusionWeek: number;
  daysForNextTraining: number;

  constructor(playerId: number) {
    this.playerId = playerId;
    this.name = '';
    this.label = '';
    this.age = 17;
    this.days = 0;
    this.form = 0;
    this.stamina = 0;
    this.leadership = 0;
    this.experience = 0;
    this.loyalty = 0;
    this.motherClubBonus = 0;
    this.specialty = 0;
    this.keeper = 0;
    this.defending = 0;
    this.playmaking = 0;
    this.winger = 0;
    this.passing = 0;
    this.scoring = 0;
    this.setPieces = 0;
    this.daysForNextTraining = 0;
    this.inclusionWeek = 1;
  }
}
